test/integration/scheduler/shoot/shoot_suite_test.go

```diff
@@ -42,10 +41,10 @@ var (
 	ctx = context.Background()
 	log logr.Logger
 
-	restConfig          *rest.Config
-	testEnv             *gardenerenvtest.GardenerTestEnvironment
-	testClient          client.Client
-	versionedTestClient *gardencoreclientset.Clientset
+	restConfig *rest.Config
+	testEnv    *gardenerenvtest.GardenerTestEnvironment
+	testClient client.Client
+	mgrClient  client.Reader
```

Comment:
I personally would prefer to have this moved to `test/integration/scheduler/shoot/shoot_test.go`, as else the definition and initialization are further apart (especially if it's only used in a single test file).

Definition could be moved to the most outer `Describe` block in the test file (between lines 31 and 32 as a var).

Reply:
It cannot be inside the Describe block, because the functions `createCloudProfile` etc are using it, we have other global variables declared in the `shoot_suite_test.go`, so I vote to keep it there. This is also consistent with other integration tests. WDYT?

Reply:
Okay fine with me for now for the reasons mentioned.